tests/conftest.py

```diff
@@ -6428,6 +6429,7 @@ def bobby_columnar_table_multi_batch_deterministic_data_context(
 
     # Re-enable GE_USAGE_STATS
     monkeypatch.delenv("GE_USAGE_STATS")
+    monkeypatch.setattr(AnonymizedUsageStatisticsConfig, "enabled", True)
```

Comment:
Maybe I'm missing something but this `delenv` statement did not enable usage stats - the statement added here ensures that we create a handler.

I've added this to all Alice/Bobby/Bobster/Quentin fixtures so we can determine their payloads.